src/Components/GrowthChart/GrowthChartBuilder/GrowthChartBuilder.tsx

```diff
@@ -12,8 +12,10 @@ export const GrowthChartBuilder = ({
     dataSetMetadata,
     xAxisValues,
     yAxisValues,
-    keysDataSet,
+    keysDataSet, optionsObject,
 }: ChartDataTypes) => {
+    Chart.register(CategoryScale, annotationPlugin);
```

Comment:
With Chart.js version 3 and later, scales and additions are automatically registered when imported, making it unnecessary to use the Chart.register() method.